src/lib.rs

```diff
@@ -655,6 +674,22 @@ impl fmt::Debug for AstarteSdk {
     }
 }
 
+mod utils {
+    pub fn extract_purgeproperties(bdata: &[u8]) -> Vec<String> {
+        use flate2::read::ZlibDecoder;
+        use std::io::prelude::*;
+
+        let mut bdata = bdata.to_owned();
+        bdata.drain(0..4);
+
+        let mut d = ZlibDecoder::new(bdata.as_slice());
```

Comment:
Are these 3 lines equivalent to
```
let mut d = ZlibDecoder::new(&bdata[4..]);
```
or am I missing something?

Reply:
yes